### DISCUSSION TITLE
O3DE Game Jam Aug 20th - Installer discussion

### DISCUSSION THREAD

**Discussion by OBWANDO** [o3de/sig-release#4] [General]
We have the O3DE Game jam coming up on the 20th. I'd like to get a discussion going about getting an installer that would be usable for that game jam. It wont be going up on the O3DE website as an official installer, but instead a linked installer for the O3DE Game Jam based on the new main when stab/2106 is merged in. We will post that it is still a developer preview and set expectations as well.

Lets figure out what is the easiest path for someone to use the installer to get the editor up and going, as well as what they need to get some script canvas going, a gem started, and other simple related items.

**Comment by ppinfel** [comment]
somewhat side note but teaching how to package/export game is gonna be important too

another thing is how to actually load levels on game start, maybe actually provide a gem because i don't think its possible without going into c++

**Reply by WashedUpStudio** [reply]
yes a gem for this would help people for a game jam.

**Reply by AMZN-alexpete** [reply]
You can create an autoexec.cfg file in your project folder with the LoadLevel command to automatically load a level:
`LoadLevel <your level name without any extension>`

**Reply by AMZN-alexpete** [reply]
At present you cannot package/export the game from the installer build, you need to do it from a full GitHub engine build, but we're looking into fixing that.
